src/back/spv/instructions.rs

```diff
@@ -454,6 +454,54 @@ pub(super) fn instruction_function_call(
 // Conversion Instructions
 //
 
+pub(super) fn instruction_convert_f_to_u(
```

Comment:
similarly to feedback of the other PR, maybe it would help to have `instruction_unary_op` to contain all the code that's repeated between these methods?

Reply:
Yes I looked at the `instruction_unary_op`, and that is possible, so I added it!